lib/src/matomo.dart

```diff
@@ -352,6 +352,15 @@ class MatomoTracker {
   @visibleForTesting
   Future<String?> getUserAgent({
     DeviceInfoPlugin? deviceInfoPlugin,
+  }) async {
+    String? result =
+        await _getUserAgentInner(deviceInfoPlugin: deviceInfoPlugin);
+    return result != null ? Uri.encodeComponent(result) : null;
```

Comment:
Prefer the syntax:
```dart
final result = await _getUserAgentInner(deviceInfoPlugin);
return result != null ? Uri.encodeComponent(result) : null;
```

Reply:
Thanks for these guides, i have reorganized the code.